### ISSUE TITLE
[2021 fall] ICCV 2021 Standardized Max Logits - A Simple yet Effective Approach for Identifying Unexpected Road Obstacles in Urban-Scene Segmentation (20204544)

### ISSUE THREAD

**Issue by psh01087** [awesome-davian/awesome-reviews-kaist#149]

**Comment by hsrhee** [comment]
안녕하세요, 논문 요약 감사합니다.

**Method**

* Iterative Boundary Suppresion에서 "에대한 receptive를 의미하고" 오타 부분 공유드립니다.

**Motivation**

* Motivation과 Method의 index가 중복된 부분 공유드립니다. (4.Method로 수정 필요)

**Conclusion**

* "모델의 output인 Max Logit의 분포의 의존하게 됩니다." 오타 부분 공유드립니다.

Conclusion에서 "inference time과 memory에 overhead가 적습니다."에 대한 실험 결과가 추가되면 이해하기에 더 좋을 것 같습니다.
(Metrics measured with the image size of 2048 × 1024 on NVIDIA GeForce RTX 3090 GPU)

**Comment by khslily98** [comment]
안녕하세요.
작성해주신 글 잘 읽었습니다. 아래는 몇가지 보완하시면 좋을 것 같은 점 들입니다.

1. 수식이 깨진 부분이 아쉽습니다. 수식을 이미지나 깨짐 없이 넣어주신다면 전반적인 식 이해가 훨씬 수월하게 될 것 같습니다.
2. conclusion 부분에서 "저희의 방법론은 도로 주행 중 unexpected obstacle을 찾아내기 위한 간단하면서 효과적인 방법론을 제시합니다. 저희의 방법론은 inference time과 memory에 overhead가 적습니다. 더 나아가, 저희의 방법론은 기존의 다른 방법론들과 상호 보완적으로 동작할 수 있습니다. " 라고 언급하는데, 그 '저희의 방법론'이 무엇 인지에 대한 설명, 즉 핵심 기법이 빠져있는 것 같습니다. Standardized Max Logits (SML)과 Iterative Boundary Suppression 등의 내용을 다시 한번 요약 정리해주셨으면 합니다.
3. SML의 구체적으로 어떤 부분이 unexpected obstacle 탐지에 효과가 준 것인지 나타내주셨으면 좋겠습니다. - 전반적으로 한편의 논문을 보는 듯한 느낌을 주는데, 리뷰에 포커스를 맞추어 강조될 부분의 내용 위주로 내용을 강조/축약해주시면 더 좋은 리뷰가 될 듯 합니다.
4. Prelinimary 에서 등장하는 OoD 가 무엇의 줄임말 인지 나타내주셨으면 좋겠습니다.

감사합니다, 남은 학기 함께 화이팅해요!😊

**Comment by ju0209** [comment]
안녕하세요, 좋은 설명 감사드립니다!

1. Method의 Iterative Boundary Suppression에서 BAP에 관해 질문이 있습니다.
   * 논문을 보니 boundary mask가 segmentation network에서 얻어진다고 되어있는데, boundary는 초록 테두리로 표시된 부분이며, 업데이트되는 부분(초록색으로 색칠된 부분)에 대한 정보도 같이 구해지는 건가요?
   * 기존 pooling의 경우에는 연산 후 크기가 줄어드는데, 그림에서 BAP 부분을 보면 현재 픽셀(노란 상자 부분)에 대해서만 값이 변경되며 크기 변화도 없는 것을 확인하였습니다. (3x3 -> 3x3 ) 그렇다면 이 과정의 이름을 pooling으로 붙이신 이유도 궁금합니다.
   * 또한 BAP의 수식에서는 한 patch에 해당하는 hxw개의 값들을 사용하여 시그마를 통해 결국 하나의 결과값을 구하는 것 같은데, patch를 boundary mask 내에서 업데이트 되는 부분의 픽셀들에 위치하게 한 뒤 그 위치에서 이 수식이 적용되어 현재 위치의 픽셀 값에 반영하는 것인가요? 정확한 과정이 어떻게 되는지 보충설명을 해주시면 더 이해가 수월할 것 같습니다.

**Typo**

* Method 파트부터 넘버링이 하나씩 밀려야할 것 같습니다.
* Method 두번째 문단 'class의 개서 C' -> 개수
* Experimental setup의 첫 번째 데이터셋 설명에서 'unexptected' -> 'unexpected'

감사합니다!!
